include/odp/api/spec/packet.h

```diff
@@ -1990,6 +1990,22 @@ int8_t odp_packet_shaper_len_adjust(odp_packet_t pkt);
  */
 void odp_packet_shaper_len_adjust_set(odp_packet_t pkt, int8_t adj);
 
+/**
+ * Get mark value from the odp_packet_t
+ *
+ * Retrieve the mark value set by classification subsystem.
```

Comment:
As I commented earlier, the logic of packet marking should be explained in the classification API and here we only need to tell that this metadata exists, has certain default value and can be retrieved through this function. Thus, I would rephrase like this:

"Get the value of the CLS mark of the packet. The mark value is zero by default but may be changed by the classification subsystem."

Reply:
Will be fixed in next version.